xmtp_proto/src/types/topic.rs

```diff
@@ -106,14 +117,20 @@ impl Topic {
         &self.inner[1..]
     }
 
-    pub fn bytes(&self) -> Vec<u8> {
-        self.inner.clone()
+    /// get the full topic bytes, including the identifying [`TopicKind`]
+    pub fn cloned_bytes(&self) -> Vec<u8> {
+        self.inner.clone().to_vec()
     }
 
+    /// consume this [`Topic`] into its bytes as a Vec
     pub fn to_bytes(self) -> Vec<u8> {
-        self.inner
+        self.inner.to_vec()
     }
```

Comment:
Why not return the `SmallVec` here? It's got almost the same api and can be convert to a vec if really needed.